joomla/joomla-cms
# [category modal field] Adds Create Category button and other improvements

Pull Request for Improvement.
### Summary of Changes

This PR is the sequence of https://github.com/joomla/joomla-cms/pull/11830 but for now for categories modal.

For list of changes see https://github.com/joomla/joomla-cms/pull/11830 (same changes, but for categories now).

Similiar PR: 
- Article: https://github.com/joomla/joomla-cms/pull/11830
- Category: https://github.com/joomla/joomla-cms/pull/11857 (this one)
- Contact: https://github.com/joomla/joomla-cms/pull/11858
- Newsfeed: https://github.com/joomla/joomla-cms/pull/11862
### Animated Gif (click to view in full screen)

![modal-category](https://cloud.githubusercontent.com/assets/9630530/18129585/45baa3fe-6f83-11e6-8968-faf306168207.gif)
### Testing Instructions
- Use latest staging
- Apply patch https://github.com/joomla/joomla-cms/pull/11830 and then this patch
- In a multilanguage install edit a category and try the modal button in the associations tab (as animated gif)
- Test all the "Edit", "Create", "Select" and "Clear" buttons.
- Code review to check all is ok.
### Documentation Changes Required

None.
